.github/workflows/dependabot-gradle.yml

```diff
@@ -0,0 +1,35 @@
+name: Fix gradle lockfile for dependabot
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  update-gradle-lock:
+    if: ${{ github.event.label.name == "dependabot-gradle" && github.event.head_commit.message != "GPC-NONE: Automatically update gradle.lockfile" }}
+    name: Fix gradle lockfile for dependabot
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT }}
```

Comment:
We should do this for the auto-format-pr stuff too if possible

Reply:
Will do that afterwards